### PR TITLE
Breakpad fixes

### PR DESCRIPTION
**Pull request by slipher**
Fix hang after crashing on Linux and unnecessary console popping up on Windows.

### CODE REVIEW

**Comment by Kangz** [comment]
LGTM

File: daemon/CMakeLists.txt [hunk 1]
```diff
@@ -571,6 +571,11 @@ if (USE_BREAKPAD)
         target_link_libraries(crash_server
             srclibs-breakpad-crash_generation_server srclibs-breakpad-common)
         target_compile_definitions(crash_server PRIVATE UNICODE)
+        if (MINGW)
```
Comment (Amanieu) [inline daemon/CMakeLists.txt]: A better way of doing this is to add `WIN32` to `add_executable`: https://cmake.org/cmake/help/v3.0/command/add_executable.html

Comment (slipher) [inline daemon/CMakeLists.txt]: Done

**Comment by DolceTriade** [comment]
LGTM.

**Pull request merged**